db/views/national_hearing_queue_entries_v02.sql

```diff
@@ -0,0 +1,43 @@
+--- this view is a way to see both of the appeals and legacy appeals information IN a single table
+--- materialized means that this information will be cached IN a temporary table
+SELECT
+    appeals.id AS appeal_id,
+    'Appeal' AS appeal_type,
+    -- COALESCE selects the first non-null value
+    COALESCE(appeals.changed_hearing_request_type, appeals.original_hearing_request_type) AS hearing_request_type,
+    CAST(appeals.receipt_date AS CHAR) AS receipt_date,
+    CAST(appeals.uuid AS CHAR) AS external_id,
+    CAST(appeals.stream_type AS CHAR) AS appeal_stream,
+    CAST(appeals.stream_docket_number AS CHAR) AS docket_number
+FROM appeals
+JOIN tasks ON tasks.appeal_type = 'Appeal' and tasks.appeal_id = appeals.id
+WHERE tasks.type = 'ScheduleHearingTask'
+  and tasks.status IN ('assigned', 'in_progress', 'on_hold')
+
+-- Union for legacy appeals equivalent of above
+UNION
+
+SELECT
+    legacy_appeals.id AS appeal_id,
+    'LegacyAppeal' AS appeal_type,
+    f_vacols_brieff.bfhr AS hearing_request_type,
+    REPLACE(CAST(f_vacols_brieff.bfd19 AS CHAR), '-', '') AS receipt_date,
```

Comment:
```suggestion
    REPLACE(CAST(f_vacols_brieff.bfd19 AS TEXT), '-', '') AS receipt_date,
```
This field is also being cut off.